CONTRIBUTING.md

```diff
@@ -50,50 +94,22 @@ remote client info, request content (with value field redacted), request
 handling latency, response size, etc. Uses zap logger if available,
 otherwise uses capnslog.
 
-Fixes #38
```

Comment:
We probably should replace it as `Ref #38`

Reply:
I think that posting issue links in commits is an overkill. Issue should be already linked in PR and github is pretty good in finding PRs from commit.